alcs-frontend/src/app/features/notice-of-intent/decision/conditions/condition/condition.component.ts

```diff
@@ -0,0 +1,104 @@
+import { AfterViewInit, Component, Input, OnInit } from '@angular/core';
+import moment from 'moment';
+import { UpdateApplicationDecisionConditionDto } from '../../../../../services/application/decision/application-decision-v2/application-decision-v2.dto';
```

Comment:
Is it correct that it is referring to Application instead of NOI?

Reply:
Dead import removed